### PR TITLE
improve description of installation of ONIE

### PR DESCRIPTION
**Pull request by DFritzsche**
* replaced non-working URL with example
* added Note about http/https URLs

### CODE REVIEW

File: setup/install_switch_image.md [hunk 1]
```diff
@@ -70,13 +70,14 @@ The switch is going to reboot and installs ONIE from the USB device.
 
 ### Update ONIE
 
-Reboot the switch. Enter the ONIE boot menu then select `ONIE: Rescue` to get into the ONIE CLI. Update ONIE with the CLI command `onie-self-update` as shown below.
+Reboot the switch. Enter the ONIE boot menu then select `ONIE: Rescue` to get into the ONIE CLI. Update ONIE given by <ONIE_URL> with the CLI command `onie-self-update` as shown in the example below.
```
Comment (KanjiMonster) [inline setup/install_switch_image.md]: Now it looks like we forgot to replace a token here. Maybe add a sentence to put the updater onto a reachable http server, and just 

> Update ONIE with the CLI command `onie-self-update` as shown in the example below.

File: setup/install_switch_image.md [hunk 2]
```diff
@@ -70,13 +70,14 @@ The switch is going to reboot and installs ONIE from the USB device.
...
 
-This example updates ONIE to the supported version on the AG7648 platform:
 ```
-onie-self-update -ev https://github.com/DeltaProducts/AG7648/blob/master/onie_image/20181109-onie-updater-x86_64-delta_ag7648-r0.bin
+onie-self-update -ev <ONIE_URL>
```
Comment (KanjiMonster) [inline setup/install_switch_image.md]: And maybe here use an actual URI that's clearly an example, e.g. `http://example.com/onie-updater`, or `http://192.0.2.123/onie-updater`, or (taken from the ONIE documentation) `http://local-http-server/onie-updater`.

Also -e is only supported on x86_64, so we should drop it from the example. Using that on AS4610 will fail.

File: setup/install_switch_image.md [hunk 3]
```diff
@@ -70,13 +70,14 @@ The switch is going to reboot and installs ONIE from the USB device.
...
 ```
 
+**Note**: The ONIE CLI command can only process http URLs and does not work with https.`
```
Comment (KanjiMonster) [inline setup/install_switch_image.md]: maybe "... only supports http ..."? Technically it also supports ftp, but we should't give people ideas ;)

Also similar to the nos install, we should have a link to the ONIE documentation here -> https://opencomputeproject.github.io/onie/cli/index.html#onie-self-update

File: setup/install_switch_image.md (hunk 1 above)
Comment (hilmarm) [inline setup/install_switch_image.md]: This sentence looks weird to me, 'update ONIE given by..'
Maybe remove the 'given by..' part, and write a sentence after the example that states
"Where <ONIE_URL> points to the switch image" or similar.

**Comment by hilmarm** [comment]
Installing ONIE from USB on the 5648 requires to change the boot order, as described in the following PDF. https://github.com/DeltaProducts/ag5648/blob/master/onie_image/ONIE_released_note_v1.00.pdf

I think we should point to it.

**Pull request merged**